libs/dh/shared/data-access-mocks/src/lib/market-participant.ts

```diff
@@ -390,21 +386,6 @@ function getAuditLogByActorId() {
   });
 }
 
-function getMarketParticipantActorActorCredentials(apiBase: string) {
-  const response: MarketParticipantActorCredentialsDto = {
-    certificateCredentials: undefined,
-    clientSecretCredentials: {
-      clientSecretIdentifier: 'client-secret-identifier-value',
-      expirationDate: '2020-09-30T12:00:00',
-    },
-  };
-
-  return http.get(`${apiBase}/v1/MarketParticipantActor/GetActorCredentials`, async () => {
-    await delay(mswConfig.delay);
-    return HttpResponse.json(response);
-  });
-}
-
```

Comment:
suggestion: add a mock for `GetActorCredentials` query

Reply:
Done in https://github.com/Energinet-DataHub/greenforce-frontend/pull/3376/commits/31088b3e2b2ef745bc167a00b82b71b36349fc59